src/plugins/intel_gna/tests/functional/shared_tests_instances/behavior/ov_plugin/core_intergration.cpp

```diff
@@ -349,6 +349,57 @@ TEST(OVClassBasicTest, smoke_SetConfigAfterCreatedFwModelPath) {
     ASSERT_EQ("model.bin", path);
 }
 
+TEST(OVClassBasicTest, smoke_InferencePrecisionExecutionModeGnaPrecisionDefaultValues) {
+    ov::Core core;
+    ov::hint::ExecutionMode execution_mode = ov::hint::ExecutionMode::UNDEFINED;
+    InferenceEngine::Precision gna_precision = InferenceEngine::Precision::UNSPECIFIED;
+
+    OV_ASSERT_NO_THROW(execution_mode = core.get_property("GNA", ov::hint::execution_mode));
+    ASSERT_EQ(ov::hint::ExecutionMode::UNDEFINED, execution_mode);
```

Comment:
how it will users what GNA returns `UNDEFINED` by default?
why not to _define_ it?

Reply:
Ok, setting ACCURACY as a default value is fine.